ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -71,7 +71,13 @@ Item {
 
                 readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
                 readonly property bool isHovered: memberItem.sensor.containsMouse
-                readonly property bool canBeBanned: !memberItem.itsMe && (model.memberRole !== Constants.memberRole.owner && model.memberRole !== Constants.memberRole.admin)
+                readonly property bool hasBanPermissions: !memberItem.itsMe && (model.memberRole !== Constants.memberRole.owner && model.memberRole !== Constants.memberRole.admin)
```

Comment:
This seems odd, what if _I_ am the owner?

Reply:
I now realise the variable name is misleading. `canBeBanned` is much better. It's basically checking if the user you want to kick is not the admin or the owner.

Reply:
Ah I see.. the name is indeed misleading; please fix :)